### PR TITLE
Decentlab

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/decentlab/ATM41/uplink.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+function consume(event) {
+  var payload = event.data.payload_hex;
+  var sample = decentlab_decoder.decode(payload);
+  var data = {};
+  var lifecycle = {};
+
+  // Default values
+  data.solar_radiation = sample["solar_radiation"];
```
Comment (mariusgiger) [inline types/decentlab/ATM41/uplink.js:186]: let's use camelCase for our object structure

File: types/decentlab/MBX/uplink.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  if (deleteUnusedKeys(data)) {
+    emit('sample', { "data": data, "topic": "default" });
+  }
+
+  if (deleteUnusedKeys(lifecycle)) {
+    emit('sample', { "data": lifecycle, "topic": "lifecycle" });
+  }
+}
+/*
+function main() {
```
Comment (mariusgiger) [inline types/decentlab/MBX/uplink.js:113]: remove this

File: types/decentlab/PR-26/default.schema.json [hunk 3]
```diff
@@ -1,23 +0,0 @@
-{
```
Comment (mariusgiger) [inline types/decentlab/PR-26/default.schema.json:1]: why delete this?

**Review by mariusgiger — CHANGES_REQUESTED** [review]
almost good, some comments

File: types/decentlab/PR-26/default.schema.json (hunk 3 above)
Comment (J2thatsme) [reply types/decentlab/PR-26/default.schema.json:1]: i don't see what it was but i guess for formatting purposes

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**